core/commands/mount_unix.go

```diff
@@ -73,74 +77,53 @@ baz
 `,
 	},
 	Options: []cmdkit.Option{
-		cmdkit.StringOption("ipfs-path", "f", "The path where IPFS should be mounted."),
-		cmdkit.StringOption("ipns-path", "n", "The path where IPNS should be mounted."),
+		cmdkit.StringOption(mountIPFSPathOptionName, "f", "The path where IPFS should be mounted."),
+		cmdkit.StringOption(mountIPNSPathOptionName, "n", "The path where IPNS should be mounted."),
 	},
-	Run: func(req cmds.Request, res cmds.Response) {
-		cfg, err := req.InvocContext().GetConfig()
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		cfg, err := cmdenv.GetConfig(env)
 		if err != nil {
-			res.SetError(err, cmdkit.ErrNormal)
-			return
+			return err
 		}
 
-		node, err := req.InvocContext().GetNode()
+		nd, err := cmdenv.GetNode(env)
 		if err != nil {
-			res.SetError(err, cmdkit.ErrNormal)
-			return
+			return err
 		}
 
 		// error if we aren't running node in online mode
-		if node.LocalMode() {
-			res.SetError(ErrNotOnline, cmdkit.ErrClient)
-			return
+		if nd.LocalMode() {
+			return err
```

Comment:
This needs to return `ErrNotOnline`. `err` is just the previous error (which should be nil).